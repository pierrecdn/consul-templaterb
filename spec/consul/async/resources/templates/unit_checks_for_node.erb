Check consul01.prod
<%
  checks_for_node('consul01.prod').each do |check|
%>-------
CheckID: <%= check['CheckID'] %>
Name: <%= check['Name'] %>
Node: <%= check['Node'] %>
Notes: <%= ERB::Util.html_escape(check['Notes']) %>
Output: <%= ERB::Util.html_escape(check['Output']) %>
Status: <%= check['Status'] %>
<% end %>