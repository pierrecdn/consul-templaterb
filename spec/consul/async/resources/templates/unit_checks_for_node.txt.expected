Check consul01.prod
-------
CheckID: serfHealth
Name: Serf Health Status
Node: consul01.prod
Notes: 
Output: Agent alive and reachable
Status: passing
